src/app/components/GalleryGrid/GalleryGrid.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import Image from "next/image";
+import { useLightbox } from "@/app/hooks/useLightbox";
+import { IconLoupe } from "@/app/icons/IconLoupe";
+import { galleryGridData } from "@/app/utils/galleryGridData";
+import Container from "../Share/Container";
+import { LightboxGallery } from "../Share/LightboxGallery";
+import { GalleryGridTitle } from "./GalleryGridTitle";
+
+export const GalleryGrid = () => {
+  const { isOpen, photoIndex, openLightbox, closeLightbox } = useLightbox();
+
+  return (
+    <section
+      id="gallery-grid"
+      className="pb-10 pt-[100px] desktop:pb-[140px] desktop:pt-[220px]"
+    >
+      <GalleryGridTitle />
+      <Container>
+        <ul className="grid grid-cols-4 gap-x-2.5 gap-y-3 desktop:gap-x-5 desktop:gap-y-[60px]">
+          {galleryGridData.map((item, index) => (
+            <li key={index}>
```

Comment:
Try making a separate component for rendering array elements. Don't forget about proper typing!